packages/kit/src/exports/vite/index.js

```diff
@@ -252,14 +252,13 @@ function kit({ svelte_config }) {
 						'$app',
 						'$env'
 					]
+				},
+				ssr: {
+					// These packages must be bundled by Kit so that we can resolve using the proper aliases and conditions
```

Comment:
I don't think it's true that we always need to bundle `@sveltejs/kit`. I expect it'd have negative performance implications. We really only need to do it if using Vitest to force it to go through the standard Vite pipeline instead of the vite-node pipeline so that `resolveId` is called

Reply:
Ok I'm back to being confused. Remind me why we need to ensure `resolveId` is called? `@sveltejs/kit` doesn't import any aliased modules. Is it about ensuring that we don't end up with two copies of `HttpError` (breaking `instanceof` checks) or something else?

Reply:
We need `resolveId` to be called to handle our virtual modules. `@sveltejs/kit` imports `__sveltekit/paths` and `__sveltekit/environment` internally

Reply:
No, it doesn't. `@sveltejs/kit` resolves to [src/exports/index.js](https://github.com/sveltejs/kit/blob/master/packages/kit/src/exports/index.js) which imports `esm-env` and [src/runtime/control.js](https://github.com/sveltejs/kit/blob/master/packages/kit/src/runtime/control.js), which doesn't import anything (but we _do_ need to ensure that those modules are only imported once for the `instanceof` checks, which is why I raise it)

Reply:
Removing it appears to do no harm — tested it with a project that links `@sveltejs/kit` and uses `redirect`, and everything is fine